week-08/JSONReader/README.md

```diff
@@ -0,0 +1,69 @@
+#  Module 8 Homework
+
+## Assignment
+
+### Part 1 - Short-answer questions
+
+a) Concurrency means doing ____________  things at once.
+
+Multiple
+
+b) In Swift's concurrency, the term "main actor" is often used in place of ____________.
+
+The main thread
+
+c) The main challenge of concurrent programming is that the ____________ of operations can change.
+
+Order
+
+d) URLSession is used to ____________ data from a URL.
+
+Fetch
+
+e) True or false: The @MainActor attribute can only be used on classes, not methods.
+
+False
+
+f) True or False: A potential concurrency problem is when multiple code paths try to access and change the same state at the same time.
+
+True
+
+g) The term often used to indicate when methods are not safe to use concurrently is ____________.
+
+Thread-unsafe
+
+h) You must provide an ____________ for a background configuration so the system can reconstruct its sessions.
+
+Identifier
+
+i) True or False: URLSession allows you to configure tasks to run in the foreground only.
+
+False
+
+j) The three concrete subclasses of URLSessionTask are ____________, ____________, and ____________.
+
+URLSessionDataTask for fetching data, URLSessionUploadTask for uploading data, and URLSessionDownloadTask for downloading data
+
+k) True or False: The OS will always strictly follow the priority you set for a task.
+
+False
```

Comment:
False. While priority hints are used, the OS my still consider other factors when scheduling tasks.